### DISCUSSION TITLE
places names in mariadb

### DISCUSSION THREAD

**Discussion by ippocratis** [photoprism/photoprism#3663] [Q&A]
i want to add a place name to multile photos at once
i know that  batch editing is planed and a ull request is already open for it
in the mean time i thought it will be quite doable with some mysql manual editing but i cant figure it out

opened a terminal in mariadb
` docker exec -it photoprism-mariadb-1 mysql -s photoprism -uroot -prootpass`
listed tables
`SHOW TABLES;`
gussed the appropriate table is photos and listed its columns
`SHOW COLUMNS FROM photos;`
i see a place_id but its type is varbinary 
cast it makes no diference
`SELECT  cast(place_id as CHAR) FROM photos;`

so

where does place names are stored in the database?

thanks

**Comment by ippocratis** [comment]
Found the place_id filtering photos from a specific day (all photos taken from the same place on that day)

`SELECT * FROM  photos WHERE photo_year = '2012' AND  photo_month = '6' AND photo_day = '27';`

Found that specific place_id as "id" column in "places" table

The places table have 

updated_at
created_at
photo_count
place_favorite
place_keywords
place_country
place_state
place_city
place_district
place_label
id

Columns

Updated the proper values e.g.

`UPDATE places SET place_label  = 'abcdefg' where id = 'yy:xxxxxx';`

**Reply by graciousgrey** [reply]
If you are looking for something in the database again, you can find the database schema in our docs :) https://docs.photoprism.app/developer-guide/database/schema/

**Reply by ippocratis** [reply]
> If you are looking for something in the database again, you can find the database schema in our docs :) https://docs.photoprism.app/developer-guide/database/schema/

Well
that was handy.
Thanks.
I wish I searched the documentation first (lol)
r.t.f.m
